Topic5/Topic5.sql

```diff
@@ -0,0 +1,80 @@
+CREATE DATABASE high_school;
+
+USE high_school;
+
+CREATE TABLE Student(
+	first_name VARCHAR(20) NOT NULL,
+	last_name VARCHAR(20) NOT NULL,
+	registration_number INT UNSIGNED NOT NULL,
+	date_of_birth DATE NOT NULL,
+	
+	INDEX(first_name,last_name),
+	
+	CONSTRAINT pk_student PRIMARY KEY (registration_number)
+	) ENGINE=InnoDB;
+	
+CREATE TABLE Teacher(
+	first_name VARCHAR(20) NOT NULL,
+	last_name VARCHAR(20) NOT NULL,
+	file_number INT UNSIGNED NOT NULL,
+	date_of_birth DATE NOT NULL,
+	
+	CONSTRAINT pk_teacher PRIMARY KEY (file_number)
+	)ENGINE=InnoDB;
+	
+CREATE TABLE Course(
+	name_course VARCHAR(20) NOT NULL,
+	id_course INT UNSIGNED NOT NULL,
+	file_number INT UNSIGNED NOT NULL,
+	hours_by_week INT UNSIGNED NOT NULL,
+		
+	INDEX(name_course),
+	
+	CONSTRAINT pk_course PRIMARY KEY (id_course),
+	
+	CONSTRAINT fk_course_file_number FOREIGN KEY (file_number) REFERENCES Teacher (file_number)
+	)ENGINE=InnoDB;
+	
+CREATE TABLE Schedule_Time(
+	id_course INT UNSIGNED NOT NULL,
+	name_of_day VARCHAR(9) NOT NULL,

```

Comment:
You could use an enum here to avoid any issue with the way the name of day is inserted on the DB